Portfolio works section and about-self section

forLoop94 opened the pull request:
The content of the "About myself" section is aligned with flexbox as stated in the project requirements.

Both CSS Grid and Flexbox combined were used for layout as stated in the project requirements.

The CSS Grid was used in the main container containing all of the cards and Flexbox was used to lay out individual cards, as stated in the project requirements.

Ul and li was used for categories in "Work" section. Html, Css, and Javascript were the list items.

Ul and li was used for categories in "About-self" section. languages, Frameworks, and skills were the list items.

I followed the design as much as I could (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

I implemented button interactions.

Qoosim reviewed (changes requested):
#  *Status: Required Changes* :recycle:

Hi @forLoop94, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 

![](https://camo.githubusercontent.com/38ce8c8b213b08e2d9bb85a0b79702ad512f3910f803ec3d7917b4e0110c9044/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966)


## Required Changes ♻️

- [ ] Kindly provide all your social handles as they are on the Microverse README template.

![your-author-handles](https://user-images.githubusercontent.com/26597432/227584235-dfa18826-3f2f-48ff-b90e-b2e4d8960deb.png)


- [ ] The link to the MIT License on your README is broken at this moment. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to set it up properly.

![license](https://user-images.githubusercontent.com/26597432/227584307-1f75273b-7b70-44da-8060-45a94aec9730.png)


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

forLoop94 commented:
1. I have added my social handles to the README file.
2. The works section is now wrapped with a h2 tag.
3. All previously omitted areas in the about self section has been added. The Figma template has been strictly followed.

Thank you.

huseyinbiyik commented on index.html:
```diff
@@ -68,6 +69,124 @@ <h2>Hey There. I'm Charles.</h2>
+      <div><button class="abount-me-btn">Get my resume</button></div>
+      <img class="right-side-image-b" src="images/illustration-about-me-mobile.png" alt="right-design about-me section">
+      <img class="hor-line" src="images/Devider.png" alt="horizental line">
+      <div class="list-container">
+        <ul class="about-me-cat">
+          <li class="language">
+            Languages
+              <div class="lang-list-container">
+                <div class="language-list">Javascript</div>
+                <div class="language-list">CSS</div>
+                <div class="language-list">HTML</div>
+              </div>
+          </li>
+          <li class="frameworks">Frameworks</li>
+          <li class="skills">Skills</li>
+        </ul>
+      </div>
```
As the previous reviewer requested please complete this section with all the information provided with Figma template. The goal of this project is to code a given design pixel-perfect. It needs to be an exact copy of Figma template. Kindly ensure, you added all languages, frameworks and skills on the template.
<img width="335" alt="Screenshot 2023-03-25 at 12 37 31 AM" src="https://user-images.githubusercontent.com/25887253/227646859-7f9a64c8-f4fe-4252-b463-a6c297303d2a.png">

huseyinbiyik reviewed (changes requested):
Hi @forLoop94 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![tenor](https://user-images.githubusercontent.com/25887253/205415161-00e79d1b-af68-4f10-9b2c-265f460dc5da.gif)

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝


 ## Required Changes ♻️
- As the previous reviewer requested, please add social links to your README. Currently, you have only a name. 
<img width="296" alt="Screenshot 2023-03-25 at 12 26 25 AM" src="https://user-images.githubusercontent.com/25887253/227644964-ee25526a-9c81-4850-8606-e5d87ba643ed.png">
You can change the links in parentheses and replace the username in brackets and it to your README file.
<img width="543" alt="Screenshot 2023-03-25 at 12 49 35 AM" src="https://user-images.githubusercontent.com/25887253/227647601-b4293ef3-fec2-44da-963a-ff7515b1012b.png">


- Your license is still broken. Kindly follow the instructions to add a license at the bottom of your README file which I marked with a red rectangle below.
<img width="819" alt="Screenshot 2023-03-25 at 12 32 36 AM" src="https://user-images.githubusercontent.com/25887253/227645651-93c4683b-80de-42a8-a850-4a65fbfc780a.png">

[OPTIONAL]After adding please delete the instructions in the red rectangle.

_Check the comments under the review._

## Optional suggestions
- Good job with the commit messages, they are descriptive and have imperative tense. But let's capitalize the first word.
<img width="543" alt="Screenshot 2023-03-25 at 12 45 33 AM" src="https://user-images.githubusercontent.com/25887253/227647271-ccdfb6f9-1e88-47c6-8f30-22cb87429c4e.png">
So, "Add button interactions..."


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

huseyinbiyik commented on index.html:
```diff
@@ -9,6 +9,7 @@
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
 </head>
 <body>
```
[OPTIONAL] There is white space on the top of your page. I think it would be better to get rid of it. 👍 Check the image below.
<img width="416" alt="Screenshot 2023-03-25 at 12 53 08 AM" src="https://user-images.githubusercontent.com/25887253/227648545-be939ef8-d571-4a65-a7e7-058b53d3a0a3.png">

forLoop94 commented:
1. I have added my social handles to the README file.
2. I have rectified the MIT LICENSE issue and I have deleted the note under it in the README file as instructed.
3. I will now write proper sentences in my commits, starting each one with an uppercase letter.
4. I have made the About-self section exactly like the screenshot from the previous reviewer(Figme template). I did not do that earlier because I don't know any frameworks or have any skills at the moment. However, the Reviewer has stated that the purpose of the project is mainly to replicate it.
5. I have removed the white space atop the headline section. I put it there to give the sticky position property of the header the needed effect. However, I have removed the space and changed the position property of the header to fixed.

Thank you.

vickymarz reviewed (approved):
Hi @forLoop94,

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Good UI design ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
